jupyter-cytoscape.json

```
@@ -0,0 +1,5 @@
+{
+  "load_extensions": {
+    "ipycytoscape/extension": true
```

```suggestion
    "jupyter-cytoscape/extension": true
```
I think this should be the extension name right?

---

Yes